app/config/AppConfig.scala

```diff
@@ -26,15 +26,19 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 @Singleton
 class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesConfig) extends FeatureSwitching {
   val auditingEnabled: Boolean = config.get[Boolean]("auditing.enabled")
-  val graphiteHost: String     = config.get[String]("microservice.metrics.graphite.host")
+  val graphiteHost: String = config.get[String]("microservice.metrics.graphite.host")
 
   def queryParametersForGetFinancialDetail(dateFrom: LocalDate, dateTo: LocalDate): String = {
-    s"?dateFrom=$dateFrom&dateTo=$dateTo&onlyOpenItems=${config.get[Boolean]("eis.onlyOpenItems")}" +
-      s"&includeStatistical=${config.get[Boolean]("eis.includeStatistical")}" +
-      s"&includeLocks=${config.get[Boolean]("eis.includeLocks")}" +
-      s"&calculateAccruedInterest=${config.get[Boolean]("eis.calculateAccruedInterest")}" +
-      s"&removePOA=${config.get[Boolean]("eis.removePOA")}" +
-      s"&customerPaymentInformation=${config.get[Boolean]("eis.customerPaymentInformation")}"
+    s"?dateFrom=$dateFrom&dateTo=$dateTo" +
+      s"&includeClearedItems=${!config.get[Boolean](" eis.includeCleared")}" +
```

Comment:
Why is this not?
```suggestion
      s"&includeClearedItems=${config.get[Boolean]("eis.includeCleared")}" +
```

Reply:
Carry over from initially using old config values, have rectified